spec/controllers/rooms_configurations_controller_spec.rb

```diff
@@ -7,6 +7,7 @@
 
   before do
     request.headers['ACCEPT'] = 'application/json'
+    create_default_permissions
```

Comment:
I think we have to add the 'CreateRoom' permission check on (most of our) endpoints and consequently add the `create_default_permissions` on (most of our) spec.